pilz_trajectory_generation/test/unittest_trajectory_generator_circ.cpp

```diff
@@ -447,12 +437,10 @@ TEST_P(TrajectoryGeneratorCIRCTest, colinearCenterDueToInterim)
  */
 TEST_P(TrajectoryGeneratorCIRCTest, centerPointJointGoal)
 {
-  // get the test data from xml
-  pilz_industrial_motion_testutils::STestMotionCommand circ_cmd;
-  circ_cmd.aux_pos_type = pilz_industrial_motion_testutils::ECircAuxPosType::eCENTER;
-  ASSERT_TRUE(tdp_->getCirc("ValidCIRCCmd2", circ_cmd)) << "failed to get circ command from test data";
-  // construct planning request
-  moveit_msgs::MotionPlanRequest req = req_director_.getCIRCJointReq(robot_model_, circ_cmd);
+  SKIP_IF_GRIPPER
```

Comment:
State why this is needed here.

Reply:
With gripper some circ tests fail, because the joint values lead to different cartesian positions. See #122